static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -43,12 +44,20 @@ const AVATAR_STYLES = {
   color: {
     size: 50,
     title: t('Default Logo'),
-    description: t('The default icon for integrations'),
+    previewText: t('The default icon for integrations'),
+    help: t('Image must be between 255px by 255px and 1024px by 1024px.'),
   },
   simple: {
     size: 20,
     title: t('Default Icon'),
-    description: t('This is an optional icon used for Issue Linking'),
+    previewText: tct('This is a silhouette icon used only for [uiDocs:UI Components]', {
+      uiDocs: (
+        <ExternalLink href="https://docs.sentry.io/product/integrations/integration-platform/ui-components/" />
+      ),
+    }),
+    help: t(
+      'Image must be between 255px by 255px and 1024px by 1024px, and may only use black and transparent pixels.'
```

Comment:
Oh it's actually 256 😬 sorry I didn't notice before. The front end error message subtracts 1 so it can say it must be larger than 255 but the range is 256 - 1024.